CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+2.15.0
+------
+* Support new cop introduced by rubocop v0.87: `Style/AccessorGrouping`, `Style/BisectedAttrAccessor`
```

Comment:
How do we feel about `Style/AccessorGrouping`? It groups blocks of `attr_reader/writer` together (see https://docs.rubocop.org/rubocop/cops_style.html#styleaccessorgrouping), but in cases where we've got a lot of them it may be better to split them logically

Reply:
I think that's an element of personal style that I'd rather avoid forcing, e.g. this should be fine:

```ruby
class Foo
  attr_reader :public_attr

  private

  attr_reader :private_attr
end
```

Reply:
I don't feel strongly here myself; if this would cause lots of churn for us then it's probably not worth it.  If it only affects a few places then might be worth turning it on.

Reply:
Alright I think I'll disable it for now, tried it in PS and I'm not convinced the proposed changes would make it any better